.clang-format

```diff
@@ -164,7 +170,7 @@ PenaltyBreakTemplateDeclaration: 10
 PenaltyExcessCharacter: 1000000
 PenaltyIndentedWhitespace: 0
 PenaltyReturnTypeOnItsOwnLine: 60
-PointerAlignment: Left
+# PointerAlignment: Left
```

Comment:
This causes all the files to be marked as misformatted right now. If we change it, we should also reformat. But probably as a separate change that we can tell github to ignore in blame.

Reply:
This change is not intentional. Will revert it.